sink/config/300-redisstreamsink.yaml

```diff
@@ -21,6 +21,7 @@ metadata:
     eventing.knative.dev/release: devel
     knative.dev/crd-install: "true"
     duck.knative.dev/addressable: "true"
+    duck.knative.dev/source: "true"
```

Comment:
A sink does not implement the source duck type.

Reply:
Oh oops, you're right. This shouldn't be added. Thanks!

Reply:
Done